core/src/main/java/com/scalar/db/storage/objectstorage/StatementHandler.java

```diff
@@ -0,0 +1,135 @@
+package com.scalar.db.storage.objectstorage;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.collect.Ordering;
+import com.scalar.db.api.ConditionalExpression;
+import com.scalar.db.api.Operation;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.common.CoreError;
+import com.scalar.db.common.TableMetadataManager;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.io.Column;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.List;
+import javax.annotation.Nonnull;
+
+public class StatementHandler {
+  protected final ObjectStorageWrapper wrapper;
+  protected final TableMetadataManager metadataManager;
+
+  @SuppressFBWarnings("EI_EXPOSE_REP2")
+  public StatementHandler(ObjectStorageWrapper wrapper, TableMetadataManager metadataManager) {
+    this.wrapper = checkNotNull(wrapper);
+    this.metadataManager = checkNotNull(metadataManager);
+  }
+
+  @Nonnull
+  protected String getNamespace(Operation operation) {
+    assert operation.forNamespace().isPresent();
+    return operation.forNamespace().get();
+  }
+
+  @Nonnull
+  protected String getTable(Operation operation) {
+    assert operation.forTable().isPresent();
+    return operation.forTable().get();
+  }
+
+  protected static void validateConditions(
```

Comment:
It looks like this method is only used in `ObjectStoragePartition`. So I think we can move it to `ObjectStoragePartition`.